docs/book/rest-api.md

```diff
@@ -1909,3 +1915,8 @@ The `expressions` field is an array of the results of evaluating each of the exp
 The `bindings` field is a JSON object mapping `string`s to JSON values that describe the variable bindings resulting from evaluating the query.
 
 If the watch was set on a data reference instead of a query, the `result` field will simply be the value of the document requested, instead of an array of values.
+
+## Methods not Allowed
```

Comment:
Let's make this a subsection of the Errors section above. Also, `Method Not Allowed` instead of `Methods not Allowed`.